### DISCUSSION TITLE
Oqtane 5 is FAST and super snappy!

### DISCUSSION THREAD

**Discussion by iJungleboy** [oqtane/oqtane.framework#3508] [Show and tell]
After upgrading blazor-cms.org to Oqtane 5 (and 2sxc 16.09) the UX got way better. 

So I wrote a brief blog about it, explaining what I think causes this impressive difference. 

https://blazor-cms.org/blog?details=oqtane-5-is-fast

enjoy :)

**Comment by sbwalker** [comment]
@iJungleboy I have noticed that Oqtane 5 on  .NET 8 is noticeably faster as well... and not only sites which are using the Blazor WebAssembly hosting model - the Blazor Server hosting model is faster as well. Sites like https://www.oqtane.org and https://blazor-cms.org are running on Blazor Server so they are not actually using Wasm (the .NET assemblies are running natively on the server - not in the browser client) but .NET 8 has significantly improved server processing as well... the full list of performance improvements in .NET 8 is extensive: https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-8/
